### DISCUSSION TITLE
Set specific build command for a domain

### DISCUSSION THREAD

**Discussion by gboutte** [vercel/community#1853] [Help] [answered]
### Summary

I have an angular application and I would like to set specific build command for each domain.

For example I have a staging.domain.com that a want to be build with 
```ng build --configuration=staging```

And my production domain.com to be build with ```ng build --configuration=production```

I did not find any parameters to do it, so did I miss it or it's not possible ?

**Comment by swarnava** [comment, marked as answer]
Hi, here is a comprehensive guide to get started: https://www.storyblok.com/tp/different-build-commands-branch-name-vercel
